index.js

```diff
@@ -176,16 +176,23 @@ Shopify.prototype.graphql = function graphql(data) {
 
   path += '/graphql.json';
 
+  const json = typeof variables !== 'undefined' && variables !== null;
+  const body = json ? JSON.stringify({
+    query: data,
+    variables
+  }) : data;
+
   const url = assign({ path }, this.baseUrl);
   const options = assign({
     timeout: this.options.timeout,
     retries: 0,
     method: 'POST',
-    body: data
+    responseType: 'json',
```

Comment:
I think this option does not exist on got@8.3.1 It should be `json` and set only when the `json` variable above is `true`.

Reply:
Isn't the response always JSON?

Reply:
Yes but afaik this option is not available on got@8.3.1 so it probably has not effect here. Makes sense?

Reply:
See, https://github.com/sindresorhus/got/tree/v8.3.2 there is no `responseType` option. Anyway on second look we try to `JSON.parse()` so we don't need this option anyway or we will end parsing twice. I think we should completely remove this option.

Reply:
That’s what I ended up doing. Will push the update shortly.

Reply:
@joelvh any way I can DM ?

Reply:
hi @alexandresaiz sent you an email.